db/migrate/20150921084620_add_timestamps_to_benefit_check.rb

```diff
@@ -0,0 +1,19 @@
+class AddTimestampsToBenefitCheck < ActiveRecord::Migration
+  def up
+    change_table(:benefit_checks) { |t| t.timestamps }
+    migrate_sql = <<SQL
```

Comment:
My comment here is to look at this way of formatting: https://github.com/bbatsov/ruby-style-guide#heredocs